src/battle_script_commands.c

```diff
@@ -6651,7 +6935,7 @@ static void atk8F_forcerandomswitch(void)
         else
         {
             valid = 0;
-            for (i = 0; i < 6; ++i)
+            for (i = 0; i < 6; i++)
```

Comment:
`PARTY_SIZE`. This function (here and in pokeemerald) is missing a bunch of `PARTY_SIZE` and `PARTY_SIZE / 2`